serverside/zerotrace/example/main.go

```diff
@@ -45,9 +84,11 @@ func webSocketHandler(w http.ResponseWriter, r *http.Request) {
 
 	clientIPstr := r.RemoteAddr
 	clientIP, _, _ := net.SplitHostPort(clientIPstr)
+	var nwLayerRttTCP, nwLayerRttICMP, nwLayerRtt0T float64
```

Comment:
I find that `time.Duration` is a more natural data type for these variables (and it allows for easy conversion between time units) but this is fine too.

Reply:
Since these are just for logging, I made them floats